src/components/customLayouts/RowLayout/RowStyles.ts

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+import { themeColor, themeSpacing } from '@amsterdam/asc-ui';
+
+const RowStyles = styled.div`
+	display: flex;
+	flex-flow: row nowrap;
+	justify-content: start;
+	align-content: start;
+	border-bottom: 1px solid ${themeColor('tint', 'level3')};
```

Comment:
voor die 1px, er bestaan geen themeUnits?

Reply:
Not currently; only themeSpacing